config/datanode.example.toml

```diff
@@ -492,12 +492,6 @@ experimental_write_cache_ttl = "8h"
 ## Buffer size for SST writing.
 sst_write_buffer_size = "8MB"
 
-## Parallelism to scan a region (default: 1/4 of cpu cores).
```

Comment:
So it's another breaking change.

Reply:
Yes. It's still compatible with the old toml file.